tools/install-arduinocli.ps1

```diff
@@ -1,8 +1,5 @@
 # Install arduino-cli
 
-mkdir "$HOME/Downloads/arduino-cli"
-curl "https://downloads.arduino.cc/arduino-cli/arduino-cli-0.2.2-alpha.preview-windows.zip" -o "$HOME/Downloads/arduino-cli/arduino-cli.zip"
+curl -s --create-dirs -o "$HOME/Downloads/arduino-cli/arduino-cli.zip" "https://downloads.arduino.cc/arduino-cli/arduino-cli-0.2.2-alpha.preview-windows.zip"
```

Comment:
Wait-wait, this is not `curl`, this is `Invoke-RestMethod` or some another ugly implementation with another set of ugly arguments.